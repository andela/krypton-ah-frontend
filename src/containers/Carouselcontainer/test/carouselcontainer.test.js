import React from 'react';
import { shallow } from 'enzyme';
import Carousel from '..';

describe('carousel test', () => {
  it('should test for the carousel container', () => {
    const wrapper = shallow(<Carousel />);
    expect(wrapper).toMatchSnapshot();
  });
});
