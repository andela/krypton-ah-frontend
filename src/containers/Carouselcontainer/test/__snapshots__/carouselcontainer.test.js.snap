// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`carousel test should test for the carousel container 1`] = `
<div>
  <index
    author="Placeholder author name"
    featuredImage="https://res.cloudinary.com/ah-krypton/image/upload/v1550007680/MainImage.png"
    title="Placeholder Article Title"
  />
</div>
`;
